utils/navigator_robotx_comms/test/set_localhost_params.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
```

Comment:
Instead of a script to set params, you can set them in the [launch/.test file](http://wiki.ros.org/roslaunch/XML/param) or [runtime](http://wiki.ros.org/Parameter%20Server#Private_Parameters)